frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -129,11 +137,27 @@ class AlertsContent extends Component {
     });
   };
 
+  handleAlertsLimitChange = (event) => {
+    this.setState({ alertsLimit: event.target.value });
+  }
+
   render() {
-    const { tab, allAlerts, unchecked, checked } = this.state;
+    const { tab, allAlerts, unchecked, checked, alertsLimit } = this.state;
     const { classes } = this.props;
     return (
       <div className={classes.root}>
+        <form align="center">
+          <InputLabel>View # of</InputLabel>
+          <TextField
+            id="alert-limit"
+            type="number"
+            label="alerts:"
+            size="small"
+            InputProps={{ inputProps: { min: 0, max: 10 } }}
```

Comment:
why are we setting up 10 for the max?

Reply:
A bit arbitrary right now, but I chose 10 because we have only generated 9 alerts max so 10 seemed reasonable. Do you think there doesn't need to be a max?